app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      include Wor::Paginate
+
+      # GET /api/v1/:user_id/rents
+      def index
+        render_paginated rents
+      end
+
+      # POST /api/v1/rents
+      def create
+        rent = Rent.create!(rent_params)
+        render json: rent
+      rescue StandardError => e
```

Comment:
You should  not rescue StandardError, but a more specific error, like the one that `create!` raises, that way if another kind of error happens, the handling should be different

Reply:
done!